docker-compose.prod.yml

```diff
@@ -0,0 +1,54 @@
+version: "2"
+services:
+
+  db:
+    extends:
+      file: ./docker/common.yml
+      service: db
+    volumes:
+      - ./data/pgdata:/var/lib/postgresql/data/pgdata
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+
+  redis:
+    image: redis:3.2.1
+    volumes:
+      - ./data/redis:/data
+
+  web:
+    build: .
+    command: uwsgi /code/listenbrainz-server/docker/uwsgi.ini
```

Comment:
I like how the prod version uses uwsgi, but the default is to use server.py. Nice!